apps/website/src/core/component/image/image.presenter.tsx

```diff
@@ -0,0 +1,12 @@
+import NextImage from 'next/image';
+import { ComponentPropsWithRef, forwardRef, ForwardRefExoticComponent } from 'react';
+
+type ImageProps = Omit<ComponentPropsWithRef<typeof NextImage>, 'sizes'> & {
```

Comment:
HTML`<img>`並びに`next/image`の`sizes`プロパティは、特にブレイクポイントによって画像の表示サイズが変化する場合において、読み込むべき最適な大きさを割り出すのに使われるとっても大切なプロパティです。
```tsx
import Image from 'next/image'
const Example = () => (
  <div className="grid-element">
    <Image
      src="/example.png"
      fill
      sizes="(max-width: 768px) 100vw,
              (max-width: 1200px) 50vw,
              33vw"
    />
  </div>
)
```
> This example sizes could have a dramatic effect on performance metrics. Without the 33vw sizes, the image selected from the server would be 3 times as wide as it needs to be. Because file size is proportional to the square of the width, without sizes the user would download an image that's 9 times larger than necessary.
https://nextjs.org/docs/api-reference/next/image#sizes より引用

**なので、絶対に上書きしないでください！**
### Refer: 
- https://nextjs.org/docs/api-reference/next/image#sizes
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-sizes